topology-machine/README.md

```diff
@@ -148,19 +148,39 @@ router twice:
  * foo <-> a
  * foo <-> a
 
-The array of routers on the right side may also include static interface assignment. Use the format `router:interface` to assign a specific interface to a link. For example:
+The array of routers on the right side may also include static interface
+assignment. Use the a dict to assign a specific interface to a link. For
+example:
 
 ```
 {
     "p2p": {
-        "foo": [ "a", "b:2" ]
+        "foo": [ "a", {"router": b, "numeric": 42} ]
```

Comment:
I picked `numeric` as  the key here because it maps to the same name later in *lltopo.json*, so users are already familiar with it.